src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.scss

```diff
@@ -54,6 +54,10 @@
     margin-right: $basic-padding * 2;
  }
 
+.dot-edit-layout__form-group-tittle{
```

Comment:
Why do we need the height for?

```.dot-edit-layout__form-group-tittle```
1. we already have the class ```form-group``` in the same element
2. Is not "title" is a template name
3. I think it should be more like: ```.dot-edit-layout__field-name``` (in case we need it at all)